content/en/docs/concepts/storage/volumes.md

```diff
@@ -529,6 +529,15 @@ See the [GlusterFS example](https://github.com/kubernetes/examples/tree/{{< para
 
 ### hostPath {#hostpath}
 
+{{< warning >}}
+There are many ways a container with unrestricted access to the host filesystem can escalate
+privileges, including reading data from other containers, and abusing the credentials of system
```

Comment:
I think it's an excellent idea to add that warning here. I'd remove the comma (,) after containers, although I see it matches how it was written in the other text. So not a big deal.